russstation/code/modules/mining/ores_coins.dm

```diff
@@ -8,6 +8,7 @@
 	item_state = "ore"
 	singular_name = "adamantine ore chunk"
 	points = 40
-	custom_materials = list(MAT_TITANIUM=MINERAL_MATERIAL_AMOUNT)

```

Comment:
This was technically my fault

Reply:
Only sort of, doesn't matter though it's fixed.